package com.niit.exception;

public class UserExistException extends Exception {
	
public UserExistException()
{
	super("User already exists");
}

	

}
